portfolio/src/main/webapp/script.js

```diff
@@ -187,3 +187,67 @@ function loadPage(pageNumber) {
   }
   page = pageNumber;  
 }
+
+function checkLogin() {
+  fetch('/login').then(response => response.json()).then(status => {
+    console.log(status);
+    const postAndDeleteElement = document.getElementById('post-and-delete');
+    postAndDeleteElement.innerHTML = '';
+    if(status[0] === 'false'){
+      const loginMessage = document.createElement('p');
+      loginMessage.innerHTML = 'To post or delete comments, please <a href="' + status[1] + '">login</a>.';
+      postAndDeleteElement.appendChild(loginMessage);
+    } else {
```

Comment:
Also from Jack: it's better to have as much of the content-building in the HTML instead of writing a lot of Javascript; so next time l'd suggest putting as much of it into the HTML as you can, but this time should be fine as-is (l think - can you have a host confirm? ) .